subprojects/plugins/src/test/groovy/org/gradle/api/plugins/JavaBasePluginTest.groovy

```diff
@@ -247,18 +248,20 @@ class JavaBasePluginTest extends AbstractProjectBuilderSpec {
         configuredJavadocTool.isPresent()
     }
 
-    void 'wires toolchain to java compile source compatibility if toolchain is configured'() {
+    void 'cannot set java compile source compatibility if toolchain is configured'() {
         given:
         def someJdk = Jvm.current()
         setupProjectWithToolchain(someJdk)
         project.java.sourceCompatibility = JavaVersion.VERSION_1_1
 
         when:
         def javaCompileTask = project.tasks.named("compileJava", JavaCompile).get()
+        javaCompileTask.sourceCompatibility
+
 
         then:
-        javaCompileTask.javaCompiler.isPresent()
-        javaCompileTask.sourceCompatibility == javaCompileTask.javaCompiler.get().metadata.languageVersion.toString()
+        def error = thrown(InvalidUserDataException)
+        error.message == 'The new Java toolchain feature cannot be used at the project level in parallel with source and/or target compatibility'
```

Comment:
Hm isn't this different than what we decided? Setting a global toolchain and indivudualc compile tasks with different target levels is something we do in gradle itself even, isn't it?

Reply:
Hm or did I misread the test. What is line 259 supposed to do? I think without it, that test makes sense

Reply:
It accesses the property, which throws. Will add a comment to clarify that.